README.md

```diff
@@ -4,92 +4,9 @@
+**Border** can create and show a border around islands which players cannot pass.
 
-##Features##
-
-* Minecraft vanilla world border (requires WorldBorderAPI plugin - see below)
-* Alternative barrier blocks border - can block transit or just show where the border is using particles.
-* Option to show maximum border - uses 🚫 particle.
-
-## Installation
-1. Put this jar into the BentoBox addons folder
-2. Restart
-
-## Configuration
-
-The config.yml file contains a number of options.
-
-### Disabled GameModes
-By default, Border will operate in all game mode worlds on the BentoBox server. To disable a game mode it is necessary to write its name on new line that starts with -. Example:
-```
- disabled-gamemodes:
-   - BSkyBlock
-```
-
-### Use WorldBorderAPI (WBAPI)
-If you want to use the vanilla world border then you must download the WorldBorderAPI plugin. You can find them here: https://github.com/yannicklamprecht/WorldBorderAPI/releases
-
-Players cannot exit past the vanilla world border, so it will completely block movement outside of a player's protected island area. If you do not want this, then do not use WBAPI.
-
-To activate WBAPI, set this to true in the config.yml:
-
-```
-use-wbapi: true
-```
-
-### Use barrier blocks.
-Only applies if WBAPI isn't used.
-
-If true, the the border will use barrier blocks to prevent most players from exiting the border. If players do manage to exit it, they will get teleported back inside it. 
-
-If false, the border is indicated by particles only.
-
-The default is to use barrier blocks.
-
-```
-use-barrier-blocks: true
-```
-
-### Default border behavior
-Players can turn the border on and off if they have the right permission using the border command. This setting makes the default on or off:
-
-``` 
-show-by-default: true
-```
-
-### Show max-protection range border.
-
-Only applies if WBAPI isn't used.
-
-This is a visual border only and not a barrier. It displays the 🚫 particle. This is useful for game modes like Boxed where the player's protection area can move around.
-
-```
-show-max-border: true
-```
-
-### Show particles
-Only applies if WBAPI isn't used.
-
-Enables/disables all types of wall particles shown by the addon. 
-
-```
-show-particles: true
-```
-
-## Commands
-
-### border
-**Command**: `/[player command] border`  
-**Description**: Turns the border on/off.  
-**Permission**: `[gamemode].border.toggle`. Default: `op`.  
-**Notes**: Since Version 3.0.0 it requires a permission.  
-
-### border type {...}
-**Command**: `/[player command] border type {barrier | vanilla}`  
-**Description**: Sets the border type.  
-**Permission**: `[gamemode].border.set-type`. Default: `true`.  
-**Example**: `/[player command] border type barrier`  
+**See the full documentation [here](https://docs.bentobox.world/en/latest/addons/Border/).**
```

Comment:
This is fine although it does place a dependency on the docs website being up. I'm okay with avoiding duplication. As a result - a comment on the audience - the docs are for admins who will use the Addon. The readme is probably best targeted at programmers and GitHub viewers so in the future things relevant to that audience can go here.